docs/csi-debug.md

```diff
@@ -0,0 +1,136 @@
+## CSI driver troubleshooting guide
+
+&nbsp;
+
+### Case#1: volume create/delete issue
```

Comment:
How CSI user can identify it was a create/delete issue or mount/unmount issue? Will the kubctl returns the error?

Reply:
If it was a create/delete issue. User will observe PV cannot be created/deleted. And POD is not scheduled. If it was a mount/umount issue. User will observe POD fail to start. And POD is scheduled.